latest/swarm/google/cloud-deployment.yaml

```diff
@@ -30,6 +30,9 @@ resources:
             user: # postgres username
             password: # postgres password
             database: # postgres database for airflow
+          grafana:
+            user: # grafana username
+            password: # grafana password
```

Comment:
Are these fields required? I haven't experimented much with this, but it would be helpful to tag the username/pw fields that are required and not with comments.

Reply:
Yes, I think all fields are requires. Not sure if I understand you, are you suggesting something like this:
```
grafana:
  # grafana username
  user: airflow
  # grafana password
  password: airflow
```

Reply:
I meant for other fields too. Something like:
```
...
postgres:
  user:  # postgres username (reqd)
  password:  # postgres password (reqd)
grafana:
  user:  # grafana username (reqd)
  password:  # grafana password (reqd)
```

Reply:
I see, yes, I think it is a good idea, should do it in a separate pr.
There are two alternative approaches I am considering, one is adding some code to set the default username/password in case they are missing, and making most of the parameters optional, (could be a potential security risk if people decided to be lazy). The other is to somehow making services like grafana optional, select different compose files based on whether grafana related info is given.

Reply:
This PR made me think of this issue, but I didn't mean to give the impression that I was requesting this change now. Feel free to merge whenever you're ready.

Reply:
Sure, to make the metrics services optional needs changes in the seuron repo, I think it is best to do that after merging the two repos